feconf.py

```diff
@@ -840,6 +856,8 @@ def get_empty_ratings():
 IMPROVEMENTS_URL_PREFIX = '/improvements'
 IMPROVEMENTS_HISTORY_URL_PREFIX = '/improvements/history'
 IMPROVEMENTS_CONFIG_URL_PREFIX = '/improvements/config'
+INCOMING_ANDROID_FEEDBACK_REPORT_URL = (
```

Comment:
Please add a comment above this URL: "Please do not change this URL without first coordinating with the Oppia Android team."

Better still, perhaps put this constant in android_validation_constants.py so that @BenHenning codeowns it?

Reply:
Agree, moved to `android_validation_constants.py`